docs/development/tutorial/README.md

```diff
@@ -23,8 +24,14 @@
 
 ## Running the code
 
-- IDE (there are launchers checked in for convenience)
-- CLI: `./gradlew :game-app:game-headed:run`
+To build and run the TripleA program (the one you play the game with) either
+
+- use IDE target `game-headed` (there are launchers checked in for convenience)
+- execute the command: `./gradlew :game-app:game-headed:run`  (if you only want to build (but not run) the source use `./gradlew :game-app:game-headed:build` instead)
+
+*Note:* The first time you (or the IDE) run a gradle command, gradle will automatically download all required dependencies and assets.  This step needs a working internet connection.
+
+*Note:* To see a non-reducted version of the output of gradle supply the `-i`/`--info` option to the command.
```

Comment:
I'm curious what the situation was where you needed the `-i` flag, when did you it come in handy?

Reply:
On my system, gradle prints what it is doing ATM but erases that information and replaces it with a "SUCCESS 127ms" message afterwards.  While figuring out how to run tests, I was curious whether Gradle actually runs the tests I specified but the output was saying nothing in particular I could not differentiate between Gradle skipping compiling&running due to nothings changed and code was run but did not fail.

If you supply `-i` (at least I) got a list of targets run and Gradle showed why or why not it ran the target (`UP-DO-DATE`, `FROM-CACHE`, ...)